auth/src/main/java/com/firebase/ui/auth/util/ui/PreambleHandler.java

```diff
@@ -55,8 +74,9 @@ private void setPreamble(TextView textView) {
         textView.setText(mBuilder);
     }
 
-    private void setupCreateAccountPreamble() {
-        String withTargets = getPreambleStringWithTargets();
+    private void setupPreamble(@StringRes int textViewText) {
```

Comment:
nit: don't really like this method name because `setupPreamble` and `setPreamble` are so similar.

Reply:
Bump.